GLTFSDK/Source/Deserialize.cpp

```diff
@@ -181,18 +181,20 @@ namespace
         bv.bufferId = std::to_string(FindRequiredMember("buffer", v)->value.GetUint());
         bv.byteOffset = GetMemberValueOrDefault<size_t>(v, "byteOffset");
         bv.byteLength = GetValue<size_t>(FindRequiredMember("byteLength", v)->value);
-        bv.byteStride = GetMemberValueOrDefault<size_t>(v, "byteStride", 0U);
 
-        // When target is not provided, the bufferView contains animation or skin data
-        auto it = v.FindMember("target");
+        auto itByteStride = v.FindMember("byteStride");
 
-        if (it != v.MemberEnd())
+        if (itByteStride != v.MemberEnd())
         {
-            bv.target = static_cast<BufferViewTarget>(it->value.GetUint());
+            bv.byteStride = itByteStride->value.GetUint();
         }
-        else
+
+        // When target is not provided, the bufferView contains animation or skin data
+        auto itTarget = v.FindMember("target");
+
```

Comment:
Dislike adding these line breaks where the variable is used on the immediate following line.